documents/policies/external/Dues System.md

```diff
@@ -1,51 +1,58 @@
+2. WCA Delegates or Regional Organizations may allow the competition organizers or other official organization to pay the Dues on their behalf. 
+   1. It is the responsibility of the WCA Delegate(s) or Regional Organization to ensure that the invoice is paid on time, even if it will be paid by someone else.
+3. After a WCA Delegate submits the results for their competition and the results are posted by the WCA Results Team (WRT), the WCA Financial Committee (WFC) will send the Delegate or Regional Organization an invoice detailing the total Due amount owed.
+   1. WCA Delegate(s) or Regional Organization will have 30 days to pay the invoice counting from the time the invoice was sent.
+   2. If payment is not received by 30 days, the Delegate or Regional Organization is considered late on their dues payment.
+      1. Delegates or Regional Organizations who are not able to pay their Dues within 30 days should contact the WFC with a reason why payment cannot be made on time. 
          1. Invoices not paid within 60 days of the date issued may be subject to disciplinary action.
             1. On a case-by-case basis the WFC will evaluate if the Delegate is experiencing an exceptional circumstance for why payment cannot be made on time. In such cases, special accommodations may be made on a one-time basis.
             2. If Delegates do not have a valid reason for the late payment (i.e. an exceptional circumstance), then the Delegates will not be allowed to announce any new competitions or place any new gear orders as Delegate until their outstanding balance is paid.
       2. Delegates with chronically late payments will be referred to their Senior Delegates for follow-up to ensure that future Dues are received on time.
          1. Frequent late payments may result in disciplinary action.
-   3. WCA Delegates may pay the Due amount using either PayPal or Stripe.
+   3. WCA Delegates or Regional Organizations may pay the Due amount using either PayPal, bank transfer or Stripe (card payment).
       1. Details for how to make these payments will be included in the invoice sent to the Delegate.
-4. The Due amount owed for a competition will be calculated on a
-per-competitor-per-competition basis.
+4. The Due amount owed for a competition will be calculated on a per-competitor-per-competition basis.
    1. Each competition will have a per-competitor Due amount owed.
    2. The total amount owed for the competition will be the per-competitor Due amount owed multiplied by the number of competitors who participated in the competition.
-   3. For a more detailed explanation of how the Due amount owed is calculated by the WFC, please refer to Appendix A.
+   3. The per-competitor Due amount will be the higher the Due amount based competition country the Due amount based on registration fees, see Appendix A
```

Comment:
To me this is not an understandable English sentence and at least one word seems to be missing somewhere. I can deduce what is meant eventually, but this formulation is going to confuse most of the readers. Please revise and rephrase this in a more plausible way.

Reply:
I think it should say "...the higher **of** the Due amount based **on** competition country and the Due amount based on registration fees".